custom_addons/softifi_graph_module_dependency/static/description/index.html

```
@@ -0,0 +1,16 @@
+        <div>
+            <h4><p>Features:</p></h4>
+            <ul>
+                <li style="list-style:none !important;"><span style="color:green;"> &#9745;</span>&nbsp;&nbsp; Show two directions module dependencies;</li>
+                <li style="list-style:none !important;"><span style="color:green;"> &#9745;</span>&nbsp;&nbsp; Show detected modules dependecies cycles  ;</li>
+                <li style="list-style:none !important;"><span style="color:green;"> &#9745;</span>&nbsp;&nbsp; Show module exclusions.</li>
+            </ul>
+        </div>
+     </div>
+</section>
+
+</section>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </section> ]

(tag-pair)